src/main/java/de/caritas/cob/agencyservice/api/repository/agency/Agency.java

```diff
@@ -136,6 +138,17 @@ public class Agency implements TenantAware {
   @Type(type = "org.hibernate.type.NumericBooleanType")
   private boolean isExternal;
 
+  @PositiveOrZero
+  @Column(name = "age_from")
+  private Short ageFrom;
+
+  @PositiveOrZero
+  @Column(name = "age_to")
+  private Short ageTo;
+
+  @Column(name = "gender")
+  private String gender;
```

Comment:
I stumbled across "gender" when I was looking at the converter. At first sight it is not clear that there is more in the "gender" as one value, namely a list.

What do you think? Names are always great.
genders
genderValues

Reply:
Renamed to genders :) I wanted to avoid it to avoid having to prepare another migration script, but I agree, it should reflect the plurality. Fixed.

Reply:
ok, I see, did not thought about that.
We need that lot of scripts, because we are deploying quite fast to dev, right?